### PR TITLE
cloud: add doc for release-20221213

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 1]
```diff
@@ -69,82 +69,53 @@ After trying **Plaground**, you can load sample data to your TiDB Cloud cluster.
...
 
     ```sql
-    SELECT TiDB_version();
+    SHOW databases;
     ```
 
-    If you see the release version information, you are ready to use your TiDB cluster.
-
-## Step 5. Query data
+    To run the query, press **Control + Enter** or click <path d="M6.70001 20.7756C6.01949 20.3926 6.00029 19.5259 6.00034 19.0422L6.00034 12.1205L6 5.33028C6 4.75247 6.00052 3.92317 6.38613 3.44138C6.83044 2.88625 7.62614 2.98501 7.95335 3.05489C8.05144 3.07584 8.14194 3.12086 8.22438 3.17798L19.2865 10.8426C19.2955 10.8489 19.304 10.8549 19.3126 10.8617C19.4069 10.9362 20 11.4314 20 12.1205C20 12.7913 19.438 13.2784 19.3212 13.3725C19.307 13.3839 19.2983 13.3902 19.2831 13.4002C18.8096 13.7133 8.57995 20.4771 8.10002 20.7756C7.60871 21.0812 7.22013 21.0683 6.70001 20.7756Z" stroke="currentColor" stroke-width="2" stroke-linejoin="round"></path> **Run**. You can see the query log and results immediately on the bottom of the page.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-quickstart.md:90]:
```suggestion
    To run the query, press **Control + Enter** or click **Run**. You can see the query log and results immediately on the bottom of the page.
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**